source/command-monitoring/tests/find.yml

```diff
@@ -198,6 +199,57 @@ tests:
               - { $numberLong: "42" }
           command_name: "killCursors"
   -
+    description: "A successful find event with a getmore and the server kills the cursor"
+    ignore_if_server_version_less_than: "3.1"
+    operation:
+      name: "find"
+      arguments:
+        filter: { _id: { $gte: 1 }}
+        sort: { _id: 1 }
+        batchSize: 3
+        limit: 4
+    expectations:
+      -
+        command_started_event:
+          command:
+            find: *collection_name
+            filter: { _id: { $gte : 1 }}
+            sort: { _id: 1 }
+            batchSize: 3
+            limit: 4
+          command_name: "find"
+          database_name: *database_name
+      -
+        command_succeeded_event:
+          reply:
+            ok: 1.0
+            cursor:
+              id: { $numberLong: "42" }
+              ns: *namespace
+              firstBatch:
+                - { _id: 1, x: 11 }
+                - { _id: 2, x: 22 }
+                - { _id: 3, x: 33 }
+          command_name: "find"
+      -
+        command_started_event:
+          command:
+            getMore: { $numberLong: "42" }
+            collection: *collection_name
+            batchSize: 3
```

Comment:
I don't agree.  With the find/getMore commands, the driver no longer has to calculate this.  The server is tracking the limit from the original find and will do the right thing regardless of the batch size sent to getMore.